gui/designwidget.cc

```diff
@@ -331,8 +329,18 @@ void DesignWidget::updateTree()
     {
         std::lock_guard<std::mutex> lock_ui(ctx->ui_mutex);
         std::lock_guard<std::mutex> lock(ctx->mutex);
-        getTreeByElementType(ElementType::CELL)->updateCells(ctx);
-        getTreeByElementType(ElementType::NET)->updateNets(ctx);
+
+        std::vector<IdString> cells;

```

Comment:
std::transform() could yield better results due to reallocations in std::vector, but I'm not sure about this. Moreover it might be better to build these vectors before taking a mutex if there's any stuttering visible that's due to updateTree() calls.

Reply:
profile first, optimize later - there might be no need for that